db/views/materialized_patient_summaries_v03.sql

```diff
@@ -0,0 +1,416 @@
+    latest_blood_sugars.latest_blood_sugar_1_blood_sugar_value,
+    latest_blood_sugars.latest_blood_sugar_1_facility_name,
+    latest_blood_sugars.latest_blood_sugar_1_facility_type,
+    latest_blood_sugars.latest_blood_sugar_1_district,
+    latest_blood_sugars.latest_blood_sugar_1_state,
+    latest_blood_sugars.latest_blood_sugar_1_follow_up_facility_name,
+    latest_blood_sugars.latest_blood_sugar_1_follow_up_date,
+    latest_blood_sugars.latest_blood_sugar_1_follow_up_days,
+
+    latest_blood_sugars.latest_blood_sugar_2_id,
+    latest_blood_sugars.latest_blood_sugar_2_recorded_at,
+    latest_blood_sugars.latest_blood_sugar_2_blood_sugar_type,
+    latest_blood_sugars.latest_blood_sugar_2_blood_sugar_value,
+    latest_blood_sugars.latest_blood_sugar_2_facility_name,
+    latest_blood_sugars.latest_blood_sugar_2_facility_type,
+    latest_blood_sugars.latest_blood_sugar_2_district,
+    latest_blood_sugars.latest_blood_sugar_2_state,
+    latest_blood_sugars.latest_blood_sugar_2_follow_up_facility_name,
+    latest_blood_sugars.latest_blood_sugar_2_follow_up_date,
+    latest_blood_sugars.latest_blood_sugar_2_follow_up_days,
+
+    latest_blood_sugars.latest_blood_sugar_3_id,
+    latest_blood_sugars.latest_blood_sugar_3_recorded_at,
+    latest_blood_sugars.latest_blood_sugar_3_blood_sugar_type,
+    latest_blood_sugars.latest_blood_sugar_3_blood_sugar_value,
+    latest_blood_sugars.latest_blood_sugar_3_facility_name,
+    latest_blood_sugars.latest_blood_sugar_3_facility_type,
+    latest_blood_sugars.latest_blood_sugar_3_district,
+    latest_blood_sugars.latest_blood_sugar_3_state,
+    latest_blood_sugars.latest_blood_sugar_3_follow_up_facility_name,
+    latest_blood_sugars.latest_blood_sugar_3_follow_up_date,
+    latest_blood_sugars.latest_blood_sugar_3_follow_up_days,
+
+    (CASE
+         WHEN next_scheduled_appointment.scheduled_date IS NULL THEN 0
+         WHEN next_scheduled_appointment.scheduled_date > date_trunc('day', NOW() - interval '30 days') THEN 0
+         WHEN (latest_blood_pressure_1_systolic >= 180
+             OR latest_blood_pressure_1_diastolic >= 110) THEN 1
+         WHEN ((mh.prior_heart_attack = 'yes'
+             OR mh.prior_stroke = 'yes')
+             AND (latest_blood_pressure_1_systolic >= 140
+                 OR latest_blood_pressure_1_diastolic >= 90)) THEN 1
+         WHEN ((latest_blood_sugar_1_blood_sugar_type = 'random'
+             AND latest_blood_sugar_1_blood_sugar_value >= 300)
+             OR (latest_blood_sugar_1_blood_sugar_type = 'post_prandial'
+                 AND latest_blood_sugar_1_blood_sugar_value >= 300)
+             OR (latest_blood_sugar_1_blood_sugar_type = 'fasting'
+                 AND latest_blood_sugar_1_blood_sugar_value >= 200)
+             OR (latest_blood_sugar_1_blood_sugar_type = 'hba1c'
+                 AND latest_blood_sugar_1_blood_sugar_value >= 9.0)) THEN 1
+         ELSE 0
+        END) as risk_level
```

Comment:
We can remove the risk level column https://simpledotorg.slack.com/archives/CFM321423/p1664354272188359. ✂️